src/applications/mhv-landing-page/tests/e2e/confirm-email-address.cypress.spec.js

```diff
@@ -0,0 +1,149 @@
+import { rootUrl } from '../../manifest.json';
+import user from '../fixtures/user.json';
+import ApiInitializer from './utilities/ApiInitializer';
+
+const MHV_EMAIL_CONFIRMATION_DISMISSED_COOKIE =
+  'MHV_EMAIL_CONFIRMATION_DISMISSED';
+
+const clickConfirmEmail = () => {
+  cy.get(
+    'va-alert[data-testid="mhv-alert--confirm-contact-email"], va-alert[data-testid="mhv-alert--confirm-error"]',
+  )
+    .find('va-button[text="Confirm"]')
+    .shadow()
+    .find('button')
+    .click();
+};
+
+const clickEditEmailLink = () => {
+  cy.get(
+    'va-alert[data-testid="mhv-alert--confirm-contact-email"], va-alert[data-testid="mhv-alert--confirm-error"]',
+  )
+    .find('va-link[href="/profile/contact-information#contact-email-address"]')
+    .shadow()
+    .find('a')
+    .click();
+};
+
+const buildUpdateEmailResponse = (isSuccess = true) => ({
+  statusCode: isSuccess ? 200 : 400,
+  body: {
+    data: {
+      id: '',
+      type: 'async_transaction_va_profile_email_address_transactions',
+      attributes: {
+        transactionId: 'email_address_tx_id',
+        transactionStatus: isSuccess ? 'RECEIVED' : 'COMPLETED_FAILURE',
+        type: 'AsyncTransaction::VAProfile::EmailAddressTransaction',
+        metadata: [],
+      },
+    },
+  },
+});
+
+describe('MHV Email Confirmation Alert - Confirm Email', () => {
+  beforeEach(() => {
+    cy.clearCookie(MHV_EMAIL_CONFIRMATION_DISMISSED_COOKIE);
+    ApiInitializer.initializeFeatureToggle.withCurrentFeatures();
+    ApiInitializer.initializeMessageData.withNoUnreadMessages();
+
+    const userWithEmail = { ...user };
+    userWithEmail.data.attributes.vet360ContactInformation.email = {
+      emailAddress: 'user@test.com',
+    };
+    cy.login(userWithEmail);
+    cy.visit(rootUrl);
```

Comment:
Same as the other file. Need a cy.wait, another cy.find, or change the gets above to finds.